resources/views/service/create_registration.blade.php

```diff
@@ -197,6 +197,16 @@ function() {
                 });
             }
         );
+
+        // If enter is pressed, keyboard is hidden on iPad and form submit is disabled
+        $(document).on('keyup keypress', function(e) {
```

Comment:
Is it useful to have enter key in the "other collectors" hit the "+" button, or taborder next?

Reply:
yep! Will amend so that it only happens in the inputs with no direct submit